src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceGeneral3.java

```diff
@@ -25,7 +25,7 @@ void mm() {
 
     public void method2() {
         // until https://github.com/checkstyle/checkstyle/issues/13011
-        Integer a = 5;  // violation 'Distance .* is 4.'
+        Integer a = 5;  // violation 'Distance .* is 2.'
         class AClass extends Parent {
```

Comment:
Yes. But I haven't looked into that issue yet.

Reply:
Has solved `extends` and `implements`. The issue of `@Override` still exists. It may be solved by adding a case for function definition in the `calculateDistanceBetweenScopes()` function.